format_code.sh

```diff
@@ -69,7 +69,7 @@ function format_c_cpp_dir_r {
     done
     find "$1" -regex '.*\.\(c\|h\)\(pp\)?' \
         | eval "grep -E -v" "${exclude_pattern_opts}" \
-        | xargs clang-format -i
+        | xargs clang-format-12 -i
```

Comment:
As a small refactor, it might be good to use a constant for this. For example, put `CLANG_FORMAT="clang-format-12"` somewhere at the top and do `| xargs $CLANG_FORMAT -i` here.

Reply:
Sounds good to me! [0efd380](https://github.com/lcm-proj/lcm/pull/466/commits/0efd3800e161023e0be31d4a698f1b005772229b)